README.md

```diff
@@ -1,10 +1,19 @@
-# mumble-pahelper
+<p align="center"><img src = "resources/MumblePAHelper.png"/></p>
 
-Mumble Positional Audio Helper is a small tool for testing the functionality
-of positional audio plugins outside of Mumble.
+Mumble Positional Audio Helper is a small tool for testing the functionality of positional audio plugins outside of Mumble.
 
-You can use this tool to quickly see if your plugin is working as it should.
-Simply place it next to your plugin (or one level up from a /plugins folder
-and launch it.
+You can use this tool to quickly see if your plugin is working as it should.<br>
+Simply place it next to your plugin and launch it.
 
-![Screenshot of pahelper](https://github.com/mumble-voip/mumble-pahelper/raw/master/media/MumblePAHelper.png)
\ No newline at end of file
+## Default paths
+### Windows
+System directory: **C:\Program Files\Mumble\Plugins** or **C:\Program Files (x86)\Mumble\Plugins**<br>
+User directory: **%AppData%\Roaming\Mumble\Plugins**<br>
+
+### Linux
+System directory: **/usr/lib/mumble**<br>
+User directory: **~/.local/share/Mumble/Plugins**<br>
+
+## Screenshots
+![Screenshot on Windows](resources/screenshot_windows.png)
+![Screenshot on Linux](resources/screenshot_linux.png)
```

Comment:
I was thinking about whether we need two screenshots just now. But I guess its reasonable.